apis/concierge/config/v1alpha1/types_credentialissuer.go.tmpl

```diff
@@ -100,6 +116,11 @@ type ImpersonationProxySpec struct {
 	//
 	// +optional
 	ExternalEndpoint string `json:"externalEndpoint,omitempty"`
+
+	// TLS contains information about how the Concierge impersonation proxy should serve TLS.
```

Comment:
Lets document what happens when this is not provided.

Reply:
Done, see next push.